python/python/lancedb/table.py

```diff
@@ -2438,7 +2438,10 @@ async def restore(self):
         await self._inner.restore()
 
     async def optimize(
-        self, *, cleanup_older_than: Optional[timedelta] = None
+        self,
+        *,
+        cleanup_older_than: Optional[timedelta] = None,
+        delete_unverified: bool = False,
```

Comment:
Could you also add documentation below for the parameter? You can model it off of the docs here:

https://lancedb.github.io/lance/api/python/lance.html#lance.LanceDataset.cleanup_old_versions

Reply:
@wjones127 updated PR.